### ISSUE TITLE
Icon is missing in the Address Bar 

### ISSUE THREAD

**Issue by FalkHe** [BrianGilbert/xdebug-helper-for-firefox#33]
I think it got lost in one of the latest Firefox Upddates.  

I'm using  FF 94.0.2 (64-Bit) on Windows right now. 
Plugin Verison is 1.0.6.

Inside the Address Bar on the right, there's only the "Fav"-Star and "Reader View" but no XDebug Helper Icon anymore. :/

**Comment by bobonov** [comment]
Win 10 FF 96.0.2, same issue here, no icon on toolbar

**Comment by lukeholder** [comment]
Firefox 101, no icon in toolbar or draggable to toolbar.

**Comment by lukeholder** [comment]
<img width="510" alt="image" src="https://user-images.githubusercontent.com/133571/173731694-ede8b33e-3cc0-49f3-b8f2-9a06af053d0f.png">

it moved to the address bar

**Comment by FalkHe** [comment]
Maybe it should be there, but as said: it isn't.

![Screenshot 2022-06-27 171328](https://user-images.githubusercontent.com/3379206/175974333-74c52ca1-d1b7-4b48-a937-11996e4514ba.png)

**Comment by BrianGilbert** [comment]
![175974333-74c52ca1-d1b7-4b48-a937-11996e4514ba png (PNG Image, 1185 × 497 pixels) 2022-06-28 08-26-45](https://user-images.githubusercontent.com/114017/176046178-d7b4d4db-a628-40b1-919e-7482adfc224b.png)

It's displaying for me in Nightly still.. It's always been in the address bar

**Comment by FalkHe** [comment]
according to this, I'm not the only one, having this issue

https://addons.mozilla.org/en-GB/firefox/addon/xdebug-helper-for-firefox/reviews/?score=1

**Comment by FalkHe** [comment]
damn, I found the reason

I forked the code and tried to debug the Addon locally.  But i couldn't get it to work somehow. It was loaded, but not executed. 

While tying to figure out why this addon doesn't work at all, I stumbled over this bug-reports:

https://bugzilla.mozilla.org/show_bug.cgi?id=1266415
https://bugzilla.mozilla.org/show_bug.cgi?id=1266409

and found the reason for this in this sentence:

> "Never remember history" is the same as always being in private browsing. It sets browser.privatebrowsing.autostart to true.

(I always have my FF on "Never remember history")

So while not explicitly in privte-mode, "Never remember history" enables something private-mode similar and therefore disables all plugins that are not enabled for private mode. ... 🙄 

**So the solution is one of:**
a) enable the history
b) allow the Xdebug Helper extension to run private mode

Hope this helps others with the same issue.

I'll provide a pull request with this info in the README.md
